standard-hackathon-rules.md

```diff
@@ -29,14 +29,14 @@ Remember that hackathons are like marathons. Some people go to compete but most
 
 After hacking finishes, teams will show their projects each other and to the judges.
 
-You are strongly encouraged to present a demo of what you have built. Pitches or presentations are discouraged. You are not judged on the quality of your pitch or the quality of your idea. As you are judged on what you built, you'll only hurt yourself by not showing a demo.
+You are strongly encouraged to present a demo of what you have built at the current hackathon. Pitches or presentations are discouraged. You are not judged on the quality of your pitch or the quality of your idea. As you are judged on what you built, you'll only hurt yourself by not showing a demo.
 
 You are encouraged to present what you have done even if your hack is broken or you weren’t able to finish. It's okay if you didn't finish your hack—that happens all the time! Completion is only one part of the judging criteria, so you might still do well. Also, demoing is not just about the competition. It's a chance to share with others what you learned and what you tried to build—that's what hacking's all about! For being courageous enough to demo, you'll receive a special MLH "I Demoed" sticker—it doesn't matter how good the demo is! In the case that you don't have anything to demo, you can give a presentation about what you tried and what you learned. Hearing what other people learned is interesting and inspiring for other attendees.
 
 
 ## Judging Criteria
 
-Teams will be judged on these four criteria. Judges will weigh the criteria equally. During judging, participants should try to describe what they did for each criterion in their project. 
+Teams will be judged on these four criteria below if one is not set by the hackathon. Judges will weigh the criteria equally. During judging, participants should try to describe what they did for each criterion in their project. 
```

Comment:
"four criteria below if **a set of judging criteria** is not **explicitly** set by the hackathon"